src/components/ShoppingBag/ShoppingList.tsx

```diff
@@ -0,0 +1,52 @@
+import { ScrollArea } from "@/components/ui/scroll-area";
+
+import ShoppingBagIcon from "@/public/icons/shopping-bag.svg";
+
+import { CartItem } from "@/components/ShoppingBag/ShoppingListItem";
+import { type ShoppingListItem as ListItem } from "@/components/ShoppingBag/types";
+import Image from "next/image";
+
+const itemsList: ListItem[] = [
```

Comment:
I think how we should name ingredients. Because in recipes, people mostly using word "ingredients", but when people do shopping, they are using probably a "Product" word. But in code, it probably should be standarized. To discuss